src/controllers/userController.js

```diff
@@ -100,4 +100,54 @@ const logOut = (req, res) => {
   }
 };
 
-module.exports = { signUp, logIn, logOut };
+/******************************************************
+ * @UPDATEUSER
+ * @route /api/v1/updateuser
+ * @method PUT
+ * @description singUp function for creating new user
+ * @body name, email, password, confirmPassword
```

Comment:
Do we need to send `confirmPassword` if we are not doing any password confirmation? Also, `User` model doesn't have `confirmPassword` field, so we can remove it.